core/indexer/common.go

```diff
@@ -281,6 +282,84 @@ func serializeTransactions(
 	return buffSlice
 }
 
+func serializeAccounts(accounts map[string]*AccountInfo) []bytes.Buffer {
+	var err error
+
+	var buff bytes.Buffer
+	buffSlice := make([]bytes.Buffer, 0)
+	for address, acc := range accounts {
+		meta, serializedData := prepareSerializedAccountInfo(address, acc)
+		if len(meta) == 0 {
+			continue
+		}
+
+		// append a newline for each element
+		serializedData = append(serializedData, "\n"...)
+
+		buffLenWithCurrentAcc := buff.Len() + len(meta) + len(serializedData)
+		if buffLenWithCurrentAcc > txsBulkSizeThreshold && buff.Len() != 0 {
```

Comment:
maybe you can rename this const `txsBulkSizeThreshold`

Reply:
done